tools/dump_kubernetes.sh

```diff
@@ -46,6 +46,10 @@ usage() {
   error '  -m, --max-bytes          max total bytes, 0=no limit, default='${DEFAULT_MAX_LOG_BYTES}
   error '  -l, --label              if set, dump logs only for pods with given labels e.g. "-l app=pilot -l istio=galley"'
   error '  -n, --namespace          if set, dump logs only for pods in the given namespaces e.g. "-n default -n istio-system"'
+  error '  -p, --pod                if set, dump logs only for pod matching these values e.g. "-n istio-system -p istio-policy-7857db5f54-wj59d -p istio-pilot-5cb9d848b5-xx25q"'
+  error '  -c, --control-plane      if set, dump debugging information from control plane'
+  error '  -u, --cluster            if set, dump logs from cluster'
```

Comment:
This has a lot of sensitive info, maybe we should exclude this.. I guess if its off by default though then we are good

Reply:
it is off by default.